src/lib/utils/isDomElement.ts

```diff
@@ -1,5 +1,6 @@
-function isDomElement(object) {
+function isDomElement(object: any): object is HTMLElement {
   return (
+    // @ts-ignore
     object instanceof window.HTMLElement ||
```

Comment:
I don't think the `window` is really useful in that case. We can simplify the condition and remove the `@ts-ignore`. Note that the rest of the condition does not entirely satisfy the type constraint. `nodeType` greater than 0 can represent `Element`, `Text`, `Comment`, ... Which is incompatible with our type but anyway it's working.

Reply:
Indeed – I didn't want to change this condition in a minor because some users probably rely on this check.

Reply:
I think React/Preact throw an error if the container is not an Element.

Reply:
I checked online the other day and apparently the second check is for browsers not supporting `window.HTMLElement`. We can still change these checks on another iteration.